docs/source/api/apollo-server.mdx

```diff
@@ -722,169 +623,40 @@ If `query` contains more than one operation definition, you must provide this op
-const express = require('express');
-const { ApolloServer } = require('apollo-server-express');
-const {
-  ApolloServerPluginDrainHttpServer,
-  ApolloServerPluginLandingPageLocalDefault
-} = require('apollo-server-core');
-const { typeDefs, resolvers } = require('./schema');
-
-async function startApolloServer() {
-  const app = express();
-  const httpServer = http.createServer(app);
-  const server = new ApolloServer({
-    typeDefs,
-    resolvers,
-    csrfPrevention: true,
-    cache: 'bounded',
-    plugins: [
-      ApolloServerPluginDrainHttpServer({ httpServer }),
-      ApolloServerPluginLandingPageLocalDefault({ embed: true }),
-    ],
-  });
-
-  await server.start();
-
-  // Additional middleware can be mounted at this point to run before Apollo.
-  app.use('*', jwtCheck, requireAuth, checkScope);
-
-  // Mount Apollo middleware here.
-  server.applyMiddleware({ app, path: '/specialUrl' });
-  await new Promise<void>(resolve => httpServer.listen({ port: 4000 }, resolve));
-  console.log(`🚀 Server ready at http://localhost:4000${server.graphqlPath}`);
-  return { server, app };
-}
-```
-
-</MultiCodeBlock>
-
-### Options
-
-<table class="field-table">
-  <thead>
-    <tr>
-      <th>Name /<br/>Type</th>
-      <th>Description</th>
-    </tr>
-  </thead>
-
-<tbody>
-<tr class="required">
-<td>
-
-##### `app`
-
-`HttpServer`
-</td>
-<td>
-
-**Required** for `applyMiddleware` (not supported for `getMiddleware`/`getHandler`). The server middleware instance to integrate with Apollo Server.
-</td>
-</tr>
-
-
-<tr>
-<td>
-
-##### `path`
-
-`string`
-</td>
-<td>
-
-The path for Apollo Server to listen on.
-
-The default value for framework-specific packages (`apollo-server-express`, `apollo-server-fastify`, etc) is `/graphql`.
-
-The default value for `apollo-server` and serverless-specific packages (`apollo-server-lambda`, etc) is `/`.
-</td>
-</tr>
-
-
-<tr>
-<td>
-
-##### `cors`
-
-`Object` or `Boolean`
-</td>
-<td>
-
-Middleware-specific configuration options for CORS. See available options for your middleware: [express](https://github.com/expressjs/cors#cors) | [hapi](https://hapi.dev/api/?v=20.1.2#-routeoptionscors) | [koa](https://github.com/koajs/cors/)
-
-Provide `false` to remove CORS middleware entirely, or `true` to use your middleware's default configuration.
-
-The default value is `true`.
-</td>
-</tr>
-
-
 <tr>
 <td>
 
-##### `bodyParserConfig`
+##### `extensions`
 
-`Object` or `Boolean`
+`object`
 </td>
 <td>
 
-Middleware-specific configuration options for body parsing. See available options for your middleware: [express](https://github.com/expressjs/body-parser#body-parser) | [koa](https://github.com/koajs/bodyparser)
-
-Provide `false` to remove body parsing middleware entirely, or `true` to use your middleware's default configuration.
+The `extensions` object is usually used to pass request metadata within your server from one touch point to another (e.g., from a request to a plugin) during the lifecycle of a request.
```

Comment:
This is generally true about `extensions` in the system in general, but in this specific case it's just supposed to represent request extensions sent in the actual request. (Which we don't currently document in https://www.apollographql.com/docs/apollo-server/v4/workflow/requests/ ...)